### PR TITLE
test=develop, modfiy the example code of NCE and PolynomialDecay

### PR DESCRIPTION
**Pull request by silingtong123**
modify the example code of NCE and PolynomialDecay

### CODE REVIEW

**Review by jzhang533 — COMMENTED** [review]
示例代码的部分OK。文档中关于参数的说明的部分也麻烦改一下。

dim (int) - 输入的维度（一般为词嵌入的维度）。
> 词潜入？  与英文文档不一致。

- **sample_weight** (Variable, 可选) - 维度为\[batch_size, 1\]，存储每个样本的权重。每个样本的默认权重为1.0。默认值：None。

> 好像没有这么个参数

- **param_attr** (ParamAttr, 可选) - 指定权重参数属性的对象。默认值>为None，表示使用默认的权重参数属性。具体用法请参见 :ref:`cn_api_fluid_ParamAttr` 。

> 跟英文文档不一致

- **bias_attr** (ParamAttr, 可选) - 指定偏置参数属性的对象。默认值为None，表示使用默认的偏置参数属性。具体用法请参见 :ref:`cn_api_fluid_ParamAttr` 。

> 跟英文文档不一致

- **num_neg_samples** (int, 可选) - 负样本的数量。默认值：10。

> API声明里默认值是None


- **custom_dist** (float[], 可选) – float[] 类型的数据，并且它的长度为 ``num_total_classes`` 。如果采样器类别为 ``custom_dist`` ，则使用此参数。custom_dist\[i\]是第i个类别被取样的概率。默认值：None

> `float[]`是C\C++里的说法，改为python友好的说法

- **is_sparse** (bool, 可选) – 指明是否使用稀疏更新，如果为True， :math:`weight@GRAD` 和 :math:`bias@GRAD` 会变为 SelectedRows。默认值：False。

> SelectedRows是什么？需要说明一下。（英文文档有同样问题）

=============

learning_rate (Variable|float32) - 初始学习率。如果类型为Variable，>则为shape为[1]的Tensor，数据类型为float32或float64；也可以是python的
float类型。

>  对应的英文版本里写的是"It also can be set to python int number.">， 麻烦也提个PR改一下。

**Review by Heeenrrry — APPROVED** [review]
LGTM
